### PR TITLE
DCAPI: support Perl sketches; processtable,ical: new Perl sketches

### PR DESCRIPTION
**Pull request by tzz**
Support sketches written in Perl in addition to CFEngine policy.  Provide two example implementations.

### CODE REVIEW

**Comment by tzz** [comment]
@estenberg @nickanderson let me know if this looks reasonable or if I should write some better examples or support more languages.

File: sketches/utilities/ical/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Description
+Parse iCal files for events and TODOs
+
+## Dependencies
+CFEngine::sketch_template
+
+## API
+### bundle: parse_ical
+* bundle option: name = Find the available processes
```
Comment (nickanderson) [inline sketches/utilities/ical/README.md]: Find available processes?

**Comment by nickanderson** [comment]
Other than finding available processes inside of ical stuff seems ok.

File: sketches/utilities/ical/README.md (hunk 1 above)
Comment (tzz) [inline sketches/utilities/ical/README.md]: Oops.  Fixed.

**Comment by tzz** [comment]
OK, fixed and regemerated `cfsketches.json`

**Pull request merged**